python/ray/train/gbdt_trainer.py

```diff
@@ -59,9 +60,38 @@ def _convert_scaling_config_to_ray_params(
             "num_actors": int(num_actors),
         }
     )
-    ray_params = ray_params_cls(
+
+    # This should be upstreamed to xgboost_ray,
+    # but also left here for backwards compatibility.
+    if not hasattr(ray_params_cls, "placement_strategy"):
+
+        class RayParamsFromScalingConfig(ray_params_cls):
+            placement_strategy: Optional[str] = None
+            pgf_args: Optional[Tuple[Any]] = None
+            pgf_kwargs: Optional[Dict[str, Any]] = None
```

Comment:
Can we just have a single option `placement_options` that contains kwargs to pass to PGF? If we want to upstream this, we should try to keep the `RayParams` dataclass clean as it is also used without Tune